src/Day.jsx

```diff
@@ -27,7 +27,9 @@ export default class Day extends PureComponent {
     this.handleSizeChangeStart = this.handleItemModification.bind(this, 'end');
     this.handleMoveStart = this.handleItemModification.bind(this, 'both');
     this.handleDelete = this.handleDelete.bind(this);
-    this.handleMouseTargetRef = element => (this.mouseTargetRef = element);
+    this.handleMouseTargetRef = (element) => {
+      this.mouseTargetRef = element;
+    };
```

Comment:
Thanks for making this change. This is better as it avoids returning anything (which might have side-effects).